.github/workflows/checks.yaml

```diff
@@ -121,7 +121,7 @@ jobs:
       - name: Install enchant and aspell
         run: |
           sudo apt-get update
-          sudo apt-get install enchant aspell-en
+          sudo apt-get install enchant-2 aspell-en
```

Comment:
What's the change here? `enchant` is no longer maintainerd?

Reply:
It does not exists anymore in debian repository, I think all the pipeline fail because of this